### ISSUE TITLE
Basically, cannot run the script on Windows 10

### ISSUE THREAD

**Issue by orseniy99** [bellingcat/tiktok-hashtag-analysis#18]
I installed everything by following the instructions (https://github.com/bellingcat/tiktok-hashtag-analysis#pre-requisites).
Tried different commands to launch but none of them work. Throws either "no such directory" error or KeyError(key)
KeyError: 'formatters'

I am using Python 3.10.5
Below I add what the console displays.

```
    (tiktok) PS C:\Users\Arseniy\AppData\Local\Programs\Python\Python310\Scripts> python -m tiktok_hashtag_analysis --help
    Traceback (most recent call last):
      File "C:\Users\Arseniy\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
        return _run_code(code, main_globals, None,
      File "C:\Users\Arseniy\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
        exec(code, run_globals)
      File "D:\some python scripts\tiktok\lib\site-packages\tiktok_hashtag_analysis\__main__.py", line 2, in <module>
        from .file_methods import log_writer
      File "D:\some python scripts\tiktok\lib\site-packages\tiktok_hashtag_analysis\file_methods.py", line 14, in <module>
        logging.config.fileConfig(path.join(path.dirname(path.abspath(__file__)), 'logging.config'))
      File "C:\Users\Arseniy\AppData\Local\Programs\Python\Python310\lib\logging\config.py", line 72, in fileConfig
        formatters = _create_formatters(cp)
      File "C:\Users\Arseniy\AppData\Local\Programs\Python\Python310\lib\logging\config.py", line 105, in _create_formatters
        flist = cp["formatters"]["keys"]
      File "C:\Users\Arseniy\AppData\Local\Programs\Python\Python310\lib\configparser.py", line 964, in __getitem__
        raise KeyError(key)
    KeyError: 'formatters'
```
(Here I am trying to run the script directly from the place where Python is installed on the computer
C:\Users\Arseniy\AppData\Local\Programs\Python\Python310)

Maybe any ideas or recommendations why it's not working and how to get it to work?
Maybe it makes sense to try to start via WSL (Windows Subsystem for Linux)

**Comment by cutterkom** [comment]
I run into the same problem on Mac and a python venv

**Comment by msramalho** [comment]
Thanks for flagging, this issue is fixed in 1.0.3.
